packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -259,6 +252,11 @@ export interface PopoverProps {
    * Additional data that is dispatched with the tracking event.
    */
   tracking?: TrackingProps;
+  /**
+   * Displace the floating element from its core placement along specified axes
+   * More on offset: https://floating-ui.com/docs/offset
+   */
+  offsetProp?: number | { mainAxis?: number; crossAxis?: number };
```

Comment:
maybe we can call this just offset?

Reply:
Thanks for the comment @amelako. :) I thought the same, but middleware returned from floating-ui to do the offsetting is also called `offset`, so I opted for `offsetProp`. Any name sounding better than that is welcome :)

Reply:
You can rename the import:

```tsx
import { offset as offsetMiddleware } from "@floating-ui/react-dom";

// ...then later

type PopoverProps = {
  offset: number | { mainAxis?: number; crossAxis?: number };
}

// ...and finally

export default function Popover({ offset }: PopoverProps) {
  const {/* things here */} = useFloating({
    middleware: [offsetMiddleware(offset)]
  });

  return (<div>I am a Popover</div>);
}
```

Reply:
Good idea! thanks @robinmetral :) I committed the change!